gslab_scons/builders/build_anything.py

```diff
@@ -48,8 +48,8 @@ def build_anything(target, source, action, env, warning = True, **kw):
     builder = AnythingBuilder(target, source, action, local_env, warning, **builder_attributes)
     bkw = {
         'action': builder.build_anything,
-        'target_factory' : local_env.env.fs.Entry,
-        'source_factory':  local_env.env.fs.Entry,
+        'target_factory' : local_env.fs.Entry,
+        'source_factory':  local_env.fs.Entry,
```

Comment:
I assume there's a reason for this change.

Reply:
I don't think the original was ever correct. `local_env` is in itself an `env` object, so we can't call another `env` on top.

Reply:
```python
scons: Reading SConscript files ...
AttributeError: 'SConsEnvironment' object has no attribute 'env':
  File "/Users/lequan/template/analysis/SConstruct", line 50:
    SConscript('source/prepare_data/SConscript')
  File "/Users/lequan/template/config/scons/scons-local-3.0.1/SCons/Script/SConscript.py", line 614:
    return method(*args, **kw)
  File "/Users/lequan/template/config/scons/scons-local-3.0.1/SCons/Script/SConscript.py", line 551:
    return _SConscript(self.fs, *files, **subst_kw)
  File "/Users/lequan/template/config/scons/scons-local-3.0.1/SCons/Script/SConscript.py", line 256:
    call_stack[-1].globals)
  File "/Users/lequan/template/analysis/source/prepare_data/SConscript", line 28:
    env     = env)
  File "/Users/lequan/anaconda/lib/python2.7/site-packages/gslab_scons/builders/build_anything.py", line 51:
    'target_factory' : local_env.env.fs.Entry,
```